### PR TITLE
Increase SQL JobQueue.Queue column size

### PR DESCRIPTION
**Pull request by maleet**
Increased Queue name and Server Id lengths as discussed at #369

### CODE REVIEW

**Comment by odinserj** [comment]
This will only increase column sizes for new installations. For old ones nothing will be changed, because the corresponding migration script will not be executed. You should revert your changes and add a new migration to the `install.sql` file with queries to modify existing tables instead to close the issue.

**Comment by maleet** [comment]
With migration it's much better indeed.

File: src/Hangfire.SqlServer/Install.sql [hunk 1]
```diff
@@ -344,13 +344,26 @@ BEGIN
 		SET @CURRENT_SCHEMA_VERSION = 4;
 	END
 
-	/*IF @CURRENT_SCHEMA_VERSION = 4
+	IF @CURRENT_SCHEMA_VERSION = 4
 	BEGIN
 		PRINT 'Installing schema version 5';
 
+		ALTER TABLE [HangFire].[JobQueue] ALTER COLUMN [Queue] NVARCHAR (50) NOT NULL;
+		PRINT 'Modified Queue length to 50';
```
Comment (odinserj) [inline src/Hangfire.SqlServer/Install.sql]: Please provide exact column names (with table name) for messages.

**Comment by odinserj** [comment]
Okay, good, but some steps needs to be done in the crappy migration implementation:

* Update the `SET @TARGET_SCHEMA_VERSION = 4;` row in the top of the `install.sql` file.
* Update the `SqlServerObjectsInstaller` class in two places.

**Comment by odinserj** [comment]
@maleet, thank you so much! Merging!

**Pull request merged**